_playwright-tests/Integration/InstallUploadRepoContent.spec.ts

```diff
@@ -0,0 +1,129 @@
+import path from 'path';
+import { test, expect, cleanupRepositories, cleanupTemplates, randomName } from 'test-utils';
+import { RHSMClient, refreshSubscriptionManager } from './helpers/rhsmClient';
+import { runCmd } from './helpers/helpers';
+import { navigateToRepositories, navigateToTemplates } from '../UI/helpers/navHelpers';
+import { closePopupsIfExist, getRowByNameOrUrl, retry } from '../UI/helpers/helpers';
+
+const uploadRepoNamePrefix = 'Upload_Repo';
+test.describe('Install Upload Repositories on Host', () => {
+  test('Install Upload Repositories on Host', async ({ page, client, cleanup }) => {
+    await cleanup.runAndAdd(() => cleanupRepositories(client, uploadRepoNamePrefix));
+    await closePopupsIfExist(page);
+    await navigateToRepositories(page);
+
+    let regClient: RHSMClient;
+    let templateName: string;
+    let hostname: string;
+    const uploadRepoName = `${uploadRepoNamePrefix}-${randomName()}`;
+
+    await test.step('Create upload repository', async () => {
+      await page.getByRole('button', { name: 'Add repositories' }).first().click();
+      await expect(page.locator('div[id^="pf-modal-part"]').first()).toBeVisible();
+      const nameInput = page.getByPlaceholder('Enter name');
+      await nameInput.click();
```

Comment:
Do we need to click on the input first?

Reply:
Why? I checked and on other places we don't do clicking first :thinking:  Is there some issue with this specific one?